test/integration/running/test_app/test_app_restart.lua

```diff
@@ -0,0 +1,3 @@
+local fiber = require('fiber')
```

Comment:
An instance of such application will be stopped when the code is executed.
It is not suitable for testing this functionality.